recipes/nupic/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "nupic" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "048dae895ca505dddb46efb8ad062f58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - asteval ==0.9.1
+    - coverage ==3.7.1+    
+    - mock ==1.0.1
+    - ordereddict ==1.1
+    - psutil ==1.0.1
+    - pytest ==2.5.1
```

Comment:
All of the py-test packages are probably testing requirements, not run requirements.